ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/scribble/v1/scribble/README.md

```diff
@@ -46,54 +57,57 @@ JavaScript handling for dialog interaction. It is already included by its edit d
-                ELEMENT cmp-adaptiveform-scribble__controls
-                    ELEMENT cmp-adaptiveform-scribble__control-brush
-                        MODIFIER cmp-adaptiveform-scribble__button
-                    ELEMENT cmp-adaptiveform-scribble__brushlist
-                    ELEMENT cmp-adaptiveform-scribble__control-clear
-                        MODIFIER cmp-adaptiveform-scribble__button
-                    ELEMENT cmp-adaptiveform-scribble__control-geo
-                        MODIFIER cmp-adaptiveform-scribble__button
-                    ELEMENT cmp-adaptiveform-scribble__control-text
-                        MODIFIER cmp-adaptiveform-scribble__button
-                    ELEMENT cmp-adaptiveform-scribble__control-message
-                ELEMENT cmp-adaptiveform-scribble__controlpanel__controls
-                    ELEMENT cmp-adaptiveform-scribble__close-button
-                    ELEMENT cmp-adaptiveform-scribble__save-button
-        ELEMENT cmp-adaptiveform-scribble__clearsign-container
-            ELEMENT cmp-adaptiveform-scribble__clearsign-title
-            ELEMENT cmp-adaptiveform-scribble__clearsign-content
-                ELEMENT cmp-adaptiveform-scribble__clearsign-message
-                ELEMENT cmp-adaptiveform-scribble__clearsign-panel
-                    ELEMENT cmp-adaptiveform-scribble__button--secondary
-                    ELEMENT cmp-adaptiveform-scribble__button--primary
+    ELEMENT cmp-adaptiveform-scribble__label-container
+    ELEMENT cmp-adaptiveform-scribble__canvas-signed-container
+        ELEMENT cmp-adaptiveform-scribble__canvas-signed-image
+    ELEMENT cmp-adaptiveform-scribble__container
+        ELEMENT cmp-adaptiveform-scribble__header
+        ELEMENT cmp-adaptiveform-scribble__content
+            ELEMENT cmp-adaptiveform-scribble__canvases
+                ELEMENT cmp-adaptiveform-scribble__signcanvases
+                    ELEMENT cmp-adaptiveform-scribble__canvas
+                    ELEMENT cmp-adaptiveform-scribble__keyboard-sign-box
+                ELEMENT cmp-adaptiveform-scribble__geocanvas
+            ELEMENT cmp-adaptiveform-scribble__controlpanel
+                ELEMENT cmp-adaptiveform-scribble__controls
+                    ELEMENT cmp-adaptiveform-scribble__control-brush
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__brushlist
+                    ELEMENT cmp-adaptiveform-scribble__control-clear
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__control-geo
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__control-text
+                        MODIFIER cmp-adaptiveform-scribble__button
+                    ELEMENT cmp-adaptiveform-scribble__control-message
+                ELEMENT cmp-adaptiveform-scribble__controlpanel__controls
```

Comment:
This should be -controls __ is not the correct way

Reply:
so change this to cmp-adaptiveform-scribble__controlpanel-controls ?

Reply:
updated.